galette/lib/Galette/Core/Galette.php

```diff
@@ -379,7 +382,9 @@ public static function getMenus(bool $public = false): array
                                 'title' => _T("Manage statuses"),
                                 'route' => [
                                     'name' => 'entitleds',
-                                    'args' => ['class' => 'status']
+                                    'args' => ['class' => 'status'],
+                                    'aliases' => ['editEntitled'],
+                                    'sub_select' => false
```

Comment:
Entitleds may be Status or Payment Type. `aliases` do not support arguments, so impossible to select any of those.

Reply:
At least, the section they belong to is opened.